packages/cells/style/placeholder.css

```diff
@@ -37,3 +37,8 @@
   box-shadow: 0 0 2px 0 rgba(0, 0, 0, 0.25);
   background-color: var(--jp-layout-color0);
 }
+
+.jp-RawCell .jp-Placeholder span,
+.jp-CodeCell .jp-Placeholder span {
```

Comment:
Please add a new class to the `span`; if after that you still need to combine selectors (hopefully not) please use descendant combinator.

Reply:
Done! Thanks